servers/movie_writer/movie_writer.cpp

```diff
@@ -101,9 +101,13 @@ void MovieWriter::begin(const Size2i &p_movie_size, uint32_t p_fps, const String
 	// Check for available disk space and warn the user if needed.
 	Ref<DirAccess> dir = DirAccess::create(DirAccess::ACCESS_FILESYSTEM);
 	String path = p_base_path.get_basename();
+	base_path = p_base_path;
```

Comment:
I set the base_path here since though the two current movie writers do set the base_path, it's not guaranteed future implementations of new MovieWriters will set the base_path directly so I wanted to set it and add `res://` if it was a relative path.